cql3/functions/native_aggregate_function.hh

```diff
@@ -32,6 +32,10 @@ public:
     virtual bool is_aggregate() const override final {
         return true;
     }
+
+    virtual bool is_reducible() const override {
+        return true;
```

Comment:
That's too dangerous. Each aggregate that implements reducible_aggregate_function() should override it separately, so we don't accidentally add a non-reducible aggregate and fail here.

Reply:
done